logger/logger.go

```diff
@@ -0,0 +1,77 @@
+package logger
+
+import (
+	"gopkg.in/Clever/kayvee-go.v6/logger"
+)
+
+var log logger.KayveeLogger
```

Comment:
Creating your new logger shouldn't be required.  It should be okay to use `logger.KayveeLogger` directly.